cpg-language-java/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/java/JavaLanguage.kt

```diff
@@ -57,6 +57,8 @@ open class JavaLanguage :
     override val unknownTypeString = listOf("var")
     override val conjunctiveOperators = listOf("&&")
     override val disjunctiveOperators = listOf("||")
+    override val compoundAssignmentOperators =
+        setOf("+=", "-=", "*=", "/=", "%=", "<<=", ">>=", ">>>=", "&=", "|=", "^=")
```

Comment:
Is `>>>=` really an operator? and is there no `<<<=` then?

Reply:
According to https://docs.oracle.com/javase/tutorial/java/nutsandbolts/operators.html that's the case. `>>>` shifts and places a 0 in the leftmost position whereas the bit depends on sign extension for `>>`.

Reply:
Can we maybe add this url as a comment to the property? In case we are wondering again some day in the future where these operators are coming from.